tests/integrations/client/Makefile

```diff
@@ -12,37 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GO_TOOLS_BIN_PATH := $(shell pwd)/../../.tools/bin
+ROOT_PATH := ../../..
+GO_TOOLS_BIN_PATH := $(ROOT_PATH)/.tools/bin
 PATH := $(GO_TOOLS_BIN_PATH):$(PATH)
 SHELL := env PATH='$(PATH)' GOBIN='$(GO_TOOLS_BIN_PATH)' $(shell which bash)
 
 static: install-tools
 	@ gofmt -s -l -d . 2>&1 | awk '{ print } END { if (NR > 0) { exit 1 } }'
 	@ golangci-lint run ./...
-	@ revive -formatter friendly -config ../../revive.toml .
+	@ revive -formatter friendly -config $(ROOT_PATH)/revive.toml .
 
 tidy:
 	@ go mod tidy
 	git diff go.mod go.sum | cat
 	git diff --quiet go.mod go.sum
 
-test: enable-codegen
-	CGO_ENABLED=1 go test -v -tags deadlock -race -cover || { $(MAKE) disable-codegen && exit 1; }
-	$(MAKE) disable-codegen
+test: failpoint-enable
+	CGO_ENABLED=1 go test -v -tags deadlock -race -cover || { $(MAKE) failpoint-disable && exit 1; }
+	$(MAKE) failpoint-disable
 
-basic-test:
-	# skip
-
-ci-test-job: enable-codegen
-	CGO_ENABLED=1 go test -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=../../... github.com/tikv/pd/tests/client
+ci-test-job:
```

Comment:
Do we need to enable the failpoint?

Reply:
It will be enabled by the root Makefile, so we don't need to enable it here again.